compose/install.md

```diff
@@ -95,114 +93,62 @@ on Microsoft Windows Server and want to install Docker Compose.
     Invoke-WebRequest "https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\Docker\docker-compose.exe
     ```
 
-**Note**: On Windows Server 2019, you can add the Compose executable to `$Env:ProgramFiles\Docker`. Because this directory is  registered in the system `PATH`, you can run the `docker-compose --version` command on the subsequent step with no additional configuration.
+**Note**: On Windows Server 2019, you can add the Compose executable to `$Env:ProgramFiles\Docker`. Because this directory is  registered in the system `PATH`, you can run the `docker compose --version` command on the subsequent step with no additional configuration.
```

Comment:
```suggestion
**Note**: On Windows Server 2019, you can add the Compose executable to `$Env:ProgramFiles\Docker\cli-plugins`. Because this directory is used for Docker CLI plugins, you can run the `docker compose --version` command on the subsequent step with no additional configuration.
```